routes/app/app-manager.js

```diff
@@ -66,6 +66,22 @@ exports.getAppByID = function(req, res, next) {
         });
 };
 
+/**
+ * Enable an app if it is disabled and vice versa.
+ */
+exports.toggleAppByID = function(req, res, next) {
```

Comment:
@nick22891 write one function that sets the state of an application (`setState()`) that modifies the state based on system defined constants.  The constants can be stored within the common-util.js file:
```
var ACTIVE = "active", DISABLED="disabled", ..... ;
```
use those constants and refer to them in your code so any changes to the values can be centralized.